src/nl/rubensten/texifyidea/settings/TexifySettings.kt

```diff
@@ -24,9 +25,50 @@ class TexifySettings : PersistentStateComponent<TexifySettings> {
     var automaticUpDownBracket = true
     var automaticItemInItemize = true
 
+    /**
+     * internal list which stores the commands data
+     */
+    private val _labelCommands: HashMap<String, LabelingCommandInformation> =
+            hashMapOf("\\label" to LabelingCommandInformation("\\label", 1, true))
+
+    /**
+     * Verify that any key in the list has a leading backslash
+     */
+    val labelCommands: Map<String, LabelingCommandInformation>
+    get() {
+        return _labelCommands.mapKeys { it.key.addLeadingSlash() }
+    }
```

Comment:
`get()` needs an indent.

Reply:
fixed in commit 4fc09f4f03cfda54b13164fb3b1d6e31c694f878